src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import logo from "./logo.svg";
+import Social from "./Social/Social";
+import { Copyright } from "./Copyright/Copyright";
```

Comment:
import from the folder root as you have index file

Reply:
fixed